### DISCUSSION TITLE
PrimeVue.Editor setting initial value

### DISCUSSION THREAD

**Discussion by robotdevil** [primefaces/community#1475] [PrimeVue] [answered]
I've got the editor up and running and it works great except the default value in the backing model isn't shown in the editor when it loads. If I replace the p-editor with a basic textarea it all works fine and the initial value is shown on load.

Any help would be appreciated. Thanks.

```
app.component('wizard-step-welcome-text', {
    template: '#v-wizard-step-welcome-text',
    data() {
        return {
            welcomeText: '<b>Initial Value</b><br />'
        }
    }
});
```

`<p-editor v-model="welcomeText" editorStyle="height: 250px" />` - doesn't show the initial value when loaded
`<textarea v-model="welcomeText" rows="10" cols="50" />` - does show the initial value

**Comment by tugcekucukoglu** [comment, marked as answer]
Model value type of Editor is string and does not render an html tag. https://primevue.org/editor/#api.editor.props.modelValue

**Reply by robotdevil** [reply]
Thanks.

An html editor that produces html as an output should be able to open with an initial html value - is there any possibility of this being changed? If so i'll transfer this discussion to an issue.
